Makefile

```diff
@@ -78,13 +78,13 @@ IMAGE_BUILD_ARGS = --build-arg VERSION=$(VERSION) \
 	    	   --build-arg BASE_IMAGE_FULL=$(BASE_IMAGE_FULL) \
 	    	   --build-arg BASE_IMAGE_MINIMAL=$(BASE_IMAGE_MINIMAL)
 
-IMAGE_BUILD_ARGS_FULL = --target full \
+IMAGE_BUILD_ARGS_MINIMAL = --target minimal \
```

Comment:
You don't have to change this target - that would be misleading. The new default image is the minimal image. In our case, we can explicitly select the minimal image.

Reply:
You already have `IMAGE_BUILD_ARGS_MINIMAL` defined later and you are using that in the image target

Reply:
Yes, actually the only change I've made there is adding the `-t $(IMAGE_TAG) \` for `IMAGE_BUILD_ARGS_MINIMAL` ( making the minimal image as default) and for `IMAGE_BUILD_ARGS_FULL` I've changed `-t $(IMAGE_TAG)`  to `-t $(IMAGE_TAG)-full` according to this commit in the upstream repo: https://github.com/kubernetes-sigs/node-feature-discovery/pull/1027/files